ad-aws-login.sh

```diff
@@ -86,6 +86,15 @@ if ! cat "${AWS_CONFIG}" | grep -q "^\[profile ${PROFILE_NAME}\]$"; then
     exit 2
 fi
 
+PROFILE_CONFIG=$(sed -n "/${PROFILE_NAME}/,/^ *$/p" ${AWS_CONFIG})
```

Comment:
PROFILE_CONFIG="$(sed -n "/${PROFILE_NAME}/,/^ *$/p" ${AWS_CONFIG})"
to preserve newlines